lib/cylc/network/client.py

```diff
@@ -34,6 +34,9 @@
     SuiteSrvFilesManager, SuiteServiceFileError)
 
 
+CONTEXT = zmq.asyncio.Context()
```

Comment:
Maybe worth adding some comment here to explain the issue for future developers?

Reply:
Could do, this is really the correct way to use contexts, what I was doing before was strange.